internal/backend/db/dbgorm/sessions.go

```diff
@@ -110,7 +110,14 @@ func addSessionLogRecord(tx *gorm.DB, sessionID string, logr sdktypes.SessionLog
 
 func (db *gormdb) AddSessionPrint(ctx context.Context, sessionID sdktypes.SessionID, print string) error {
 	return translateError(
-		addSessionLogRecord(db.db, sessionID.String(), sdktypes.NewPrintSessionLogRecord(print)))
+		addSessionLogRecord(db.db, sessionID.String(), sdktypes.NewPrintSessionLogRecord(print)),
```

Comment:
Let's add tests? at least to check that all aspects of stopped session is working as expected?
e.g. in listDeploymentsWIthStats and other places?

Reply:
too complicated for now - will do in a future PR